rhombus/scribblings/ref-number.scrbl

```diff
@@ -210,8 +208,7 @@
   8/2
```

Comment:
This and the lower example are not exemplifying the `/` operator.

Reply:
Changed - thanks!